Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,462 @@
+  % The \textbackslash{}thanks command puts the contents inside those brackets in a footnote at the bottom of the first page. Technically speaking, \textbackslash{}thanks is just a specially formatted footnote.
+  % IEEE also has a ``long form'' author block for many authors. Check here for more information:
+  % \url{https://tex.stackexchange.com/questions/156523/multiple-authors-with-common-affiliations-in-ieeetran-conference-template}
+  % Read here for a more advanced options to modifying footnotes in the author block:  \url{http://tex.stackexchange.com/questions/826/symbols-instead-of-numbers-as-footnote-markers}
+  %   Here, we use the IEEE long-form author block.
+  \IEEEauthorblockN{% This block is for author Names.
+    James~Emerson~Parkus\IEEEauthorrefmark{1},  %the number in the bracket is a reference number to identify this footnote. \LaTeX will figure out what symbol to put there.
+    David~Breen\IEEEauthorrefmark{2}
+  }
+  \IEEEauthorblockA{% This block is for the author Afficliations, aka department and university
+    RIT Space Exploration, Rochester Institute of Technology \\ %\\ starts a new line
+    Rochester, N.Y. \\
+    Email:
+    \IEEEauthorrefmark{1}jep7631@rit.edu,
+    \IEEEauthorrefmark{2}djb1410@rit.edu
+}
+  %%   Below, we use the short-form author block and basically hack it to suit our needs.
+   %James~Emerson~Parkus$^{*\dagger}$%
+     %\thanks{$^{*}$Project Champion}%
+     %\thanks{$^{\dagger}$BS '20, Mechanical Engineering Technology}
+
+   %David~Breen$^{\ddagger}$%
+     %\thanks{$^{\ddagger}$BS '19, Electrical-Mechanical Engineering Technology}
+
+  %%   If there are many authors, consider using symbolic, numeric (aka arabic),  alphabet footnotes or a combination thereof.
+  %% the recommended order for symbolic footnotes is
+  %%   (1) asterisk        *   *
+  %%   (2) dagger          †   \dagger
+  %%   (3) double dagger   ‡   \ddagger
+  %%   (4) section symbol  §   \S
+  %%   et cetera. For higher counts, use 2x symbols (1)-(4) (i.e. (5) two asterisks **). Keep cycling through (1)-(4) using 3x, 4x, and so on.
+  %%   Note that these symbol codes work in math mode and text mode.
+  %%   There are ways to make LaTeX do this for you, but it is more advanced and not entirely necessary, especially for short author lists. Not worth the hassle, in my opinion.
+}
+% page header for pages other than cover page
+\markboth{Project Design Document Standard}%
+{Linden \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+  This project will extend off the work of the SPEX propulsion team's cold gas thruster, from the 2017 spring semester.
+  The goal of the project is to create a cold gas thruster that functions well and provides experience and data from performing a cold burn.
```

Comment:
How does this look?
`  This project concerns the design, fabrication, and experimentation of a cold gas thruster. The project's main goal is to provide insight
  and experience into the design of a supersonic nozzle for small propulsion systems. But secondary goal's will include the understanding
  in the design of high-pressure flow systems, including the necessary precautions and procedures to ensure safe usage. The propulsion subsystem
  will provide the thrust and consist of a high pressure tank, a regulator, a solenoid, and the appropriate piping and interfaces. The structure subsystem will
  will hold the propulsion system in place and allow measurement of the thrust and will consist of aluminum sheet metal and t-slotting. This project will be the first
  step in the design of a cold gas thruster for a CubeSat to allow atmospheric re-entry. `